src/main/java/org/gridsuite/directory/server/DirectoryController.java

```diff
@@ -263,4 +263,13 @@ public ResponseEntity<Page<DirectoryElementInfos>> searchElements(
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON)
                 .body(service.searchElements(userInput, directoryUuid));
     }
+
+    @GetMapping(value = "/directories/uuid", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get uuid from directory described by the path")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "The directory uuid"),
+    })
+    public ResponseEntity<UUID> getDirectoryUuidFromPath(@RequestParam("directoryPath") String directoryPath) {
```

Comment:
We should decode path here, same model as for this commit :
https://github.com/gridsuite/directory-server/commit/4d88e0df0ae99939ebf23a1645ed98bafc7f51b9

Reply:
:+1:  Done